### ISSUE TITLE
When getMyLocation is called, the map is no longer displayed after the second time?

### ISSUE THREAD

**Issue by nrikiji** [mapsplugin/cordova-plugin-googlemaps#2030]
It works
```
.controller('DashCtrl', function($scope, $timeout, $ionicPlatform) {
  $ionicPlatform.ready(function() {
    var div = document.getElementById("map");
    plugin.google.maps.Map.getMap(div);
  });
})
```

It does not work
```
.controller('DashCtrl', function($scope, $timeout, $ionicPlatform) {
  $ionicPlatform.ready(function() {
    plugin.google.maps.Map.getMap()
    .getMyLocation(function(location) {
      alert('get current position');
    });
    var div = document.getElementById("map");
    plugin.google.maps.Map.getMap(div);
  });
})
```
I tried this code on android, and the map is no longer displayed after the second time. Is this a bug?
It worked correctly on ios.

VERSION INFO
```
$ ionic plugin ls
com.googlemaps.ios 2.5.0 "Google Maps SDK for iOS"
cordova-android-play-services-gradle-release 1.2.0 "cordova-android-play-services-gradle-release"
cordova-plugin-googlemaps 2.2.0-beta-20180126-0043 "cordova-plugin-googlemaps"
・・・
```

https://github.com/nrikiji/ionic1_map2
You can check the code with this branch

**Comment by wf9a5m75** [comment]
Before asking, please test your code with the latest version always. 
`2.2.0-beta-20180126-0043` is old version.

**Comment by nrikiji** [comment]
I'm sorry...
We confirmed that it was fixed in 2.2.0 - beta - 20180127 - 0009. Thank you very much.
